lang/python/python-pynacl/patches/00-use_system_libsodium.patch

```diff
@@ -0,0 +1,18 @@
+--- ./setup.py-orig	2018-09-26 20:57:49.000000000 +0200
```

Comment:
In your commit, you haven't said anything about this patch. In past when I was looking into it, this shouldn't be necessary.

Reply:
You're right, it should not have been there.